src/b_setup.hpp

```diff
@@ -12,6 +12,10 @@ PUSH_NO_WARNINGS
 POP_NO_WARNINGS
 #endif
 
+#ifdef TEST_VERIFY_MODE
+    #include "TestMenu.hpp"
```

Comment:
Spelled wrong, needs to be `testmenu.hpp`

Reply:
Windows doesn't care about file system casing :-).